draft-ietf-jsonpath-base.md

```diff
@@ -282,6 +282,35 @@ go with a "consensus" between implementations even if it is rough, as
 long as that does not jeopardize the objective of obtaining a usable,
 stable JSON query language.
 
+## JSON Values
+
+The JSON value a JSONPath query is applied to is, by definition, a
```

Comment:
This sounds as if a JSON text were a meaningful alternative.

Reply:
Please do not *change* and *move* a section in the same PR.  This is now hard to review.